File: angular-client/src/app/Model/Country.ts
export class Country {
  public id: number;
  public description: string;
  public category: string;
  public code: string;

  constructor(id: number, description: string, category: string, code: string) {
    this.id = id;
    this.description = description;
    this.category = category;
    this.code = code;
  }
}

File: ProductsSolution/Entities/Country.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Entities
{
    public class Country : BaseEntity
    {
        public string description { get; set; }

        public string category { get; set; }

        public string code { get; set; }
    }
}
